skymap_scanner/server/start_scan.py

```diff
@@ -1059,6 +1059,18 @@ def _nside_and_pixelextension(val: str) -> int:  # -> Tuple[int, int]:
         nargs='*',
         type=_nside_and_pixelextension,
     )
+    # --real-event XOR --simulated-event
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
+        "--real-event",
```

Comment:
Could we perhaps add an assertion for `--real-event` to ensure the run id is within some range? @kjmeagher has stated `real data runs are currently between 100000 and 137579`, so perhaps we can check between `100000-999999`.

Reply:
Would it be sufficient to say `>=100000`?

Reply:
actually, anything less than 1M is probably real data?  vs simulation is in the billions

Reply:
Yea I think there needs to be an upper bound since simulation runs can be greater. A lower bound could also be good since we know data will not go below while homebrew simulation may have runids like 0-999 or so.

Reply:
since this will error loudly, we're not risking data corruption

Reply:
Yep that would be fine.